x-pack/plugin/ilm/qa/rest/build.gradle

```diff
@@ -18,6 +19,11 @@ tasks.named("yamlRestTest").configure {
   systemProperty 'tests.rest.cluster.password', clusterCredentials.password
 }
 
+tasks.named("yamlRestCompatTest").configure {
```

Comment:
For these things which are applicable to all tests suites we can use `tasks.withType(Test).configureEach { }` to avoid the duplication. If the project has internal cluster or unit test which this isn't applicable to you can use `withType(RestIntegTestTask)` as well.

Reply:
6cd87e1